reports/generator/generator.go

```diff
@@ -223,6 +236,16 @@ func (g *generator) getComplexValueFromData(spec ValueSpec, data []*map[string]i
 			return true, fmt.Sprintf("%s %s", v[0], spec.Unit)
 		}
 		return found, value
+	case TYPE_DATETIME:
+		found, value = g.getSimpleValueFromData(data, fmt.Sprintf("%s%s", prefix, spec.EhrPath))
```

Comment:
Would it be possible to turn those two return values around to make it more aligned to core golang functionality?

Example: 
```golang
someMap := map[string]string{}
value, ok := someMap["aKey"]
```

Reply:
sure

Reply:
done